helm/oncall/values.yaml

```diff
@@ -7,8 +7,8 @@ base_url: example.com
 image:
   # Grafana OnCall docker image repository
   repository: grafana/oncall
-  tag:
-  pullPolicy: IfNotPresent
+  tag: latest
```

Comment:
I wonder if it's possible to use `appVersion` from `Chart.yml` as the tag here? Would it make more sense?

Reply:
good call.

Actually I reverted `tag: latest` because I realized that the engine/celery worker container specs do this:
```yaml
image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

so by bumping the `appVersion` in this PR I'm effectively doing what you're mentioning